index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet" />
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <ul class="social-nav">
+            <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+
+    <nav>
+        <a href="index.html"><img src="img/logo.png" alt="Logo" /></a>
+        <ul class="nav-menu">
+            <li class="nav-item">
+                <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+            </li>
+        </ul>
+        <a class="nav-btn" href=""><span>CC COMPAIGN</span></a>
+        <div class="hamburger-menu">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </div>
+    </nav>
+
+    <header>
+        <div class="main-section">
+            <h2>Hello, Sharing World!</h2>
+            <h1>We are e-marketing solution agency</h1>
+            <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta doloremque officia enim adipisci sapiente expedita voluptas blanditiis animi aspernatur maxime ipsa provident quod odit beatae, ducimus excepturi explicabo aliquam architecto?
+            </p>
+        </div>
```

Comment:
- [ ] Please I would first recommend you for a semantic reason here to use a `section` tag instead of a `div` one.

- [ ] And make sure that this section includes all of these a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. Like in the design template.

## The template design 👇️
![Screenshot from 2022-10-19 20-08-13](https://user-images.githubusercontent.com/89228743/196793524-9a90a2ba-74da-4d5c-ac72-a399fce6ca58.png)

## Your design 👇️
![Screenshot from 2022-10-19 20-09-46](https://user-images.githubusercontent.com/89228743/196793782-483d633e-57e2-4b8c-a7a0-cb758b074469.png)